AEPCore/Sources/migration/V4Migrator.swift

```diff
@@ -122,7 +122,7 @@ struct V4Migrator {
 
         // save values
         let identityDataStore = NamedCollectionDataStore(name: V4MigrationConstants.Identity.DATASTORE_NAME)
-        let identityPropsData = try? JSONSerialization.data(withJSONObject: identityPropsDict)
+        let identityPropsData = AnyCodable.from(dictionary: identityPropsDict as [String : Any])
```

Comment:
Is this required? From my understanding putting the data representation of the dict should be nearly equivalent.

Reply:
Gotcha, so this is an optimization. 👍🏻